guides/micronaut-patterns-composite/micronaut-patterns-composite.adoc

```diff
@@ -0,0 +1,108 @@
+common:header-top.adoc[]
+
+common:requirements.adoc[]
+
+common:completesolution.adoc[]
+
+== Composite Pattern
+
+Imagine you want to create an API to resolve a color:
```

Comment:
The guide is great. But I miss an introduction paragraph. You jump straight to the solution of a problem/need that is worth introducing first, IMO.